website/core/TabsConstants.js

```diff
@@ -25,6 +25,12 @@ const androidLanguages = [
 ];
 const defaultAndroidLanguage = 'java';
 
+const turboModuleSpecLanguages = [
+  {label: 'Flow', value: 'flow'},
+  {label: 'TypeScript', value: 'typescript'},
+];
+const defaultTurboModuleSpecLanguage = 'flow';
```

Comment:
Just a nit here: this shouldn't be TM specific as we have Specs for Fabric components as well:

```suggestion
const javascriptSpecLanguages = [
  {label: 'Flow', value: 'flow'},
  {label: 'TypeScript', value: 'typescript'},
];
const defaultJavascriptSpecLanguages = 'flow';
```

Reply:
Good suggestion, I'll add that in now